cmd.sh

```diff
@@ -78,3 +66,13 @@ anaconda --token "${ANACONDA_TOKEN}" upload \
   ${LABEL_ARGS} \
   "${INPUT_ARTIFACTS_PATH}"/*.whl
 echo "Index: https://pypi.anaconda.org/${ANACONDA_ORG}/simple"
+
+# env cleanup
+unset INPUT_ARTIFACTS_PATH
+unset INPUT_ANACONDA_NIGHTLY_UPLOAD_ORGANIZATION
+unset INPUT_ANACONDA_NIGHTLY_UPLOAD_TOKEN
+unset INPUT_ANACONDA_NIGHTLY_UPLOAD_LABELS
+
+unset ANACONDA_ORG
+unset ANACONDA_TOKEN
+unset ANACONDA_LABELS
```

Comment:
This is my paranoia of leaking secrets accidentally, but I don't think it is actually needed.